libs/gltfio/src/AssetLoader.cpp

```diff
@@ -1226,7 +1224,7 @@ MaterialInstance* FAssetLoader::createMaterialInstance(const cgltf_data* srcAsse
     }
 
     if (matkey.hasBaseColorTexture) {
-        addTextureBinding(mi, "baseColorMap", baseColorTexture.texture, true);
+        mResult->addTextureBinding(mi, "baseColorMap", baseColorTexture.texture, true);
```

Comment:
`mResult` is kind of a weird name for a class attribute. `mAsset` maybe?

Reply:
Yes good idea, added a commit.